_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -1053,6 +1053,10 @@ private function _process_onboarding( $data ) {
 			}
 		}
 
+		if ( isset( $data['businessAddress'] ) ) {
+			self::add_business_address( $data );
```

Comment:
We still need to handle the errors in the same way that we handle them for the rest of the onboarding settings (adding them to the `error` array). So you might want to update `add_business_address()` to return either an array of errors, a `WP_Error` object, or `true` on success; and if there are any errors, merge them with `$errors`.